app/models/comment.rb

```diff
@@ -0,0 +1,9 @@
+class Comment < ApplicationRecord
+  validates :Text, presence: true
+  belongs_to :post
+  belongs_to :user
+
+  def update_comment_counter
+    post.increment!(:commentscounter)
+  end
+end
```

Comment:
- [ ] [OPTIONAL] This approach works well, however it would be nice to use `after_save` Rails callback to call the `update_comment_counter` method to increase your `commentscounter` each time a new comment is created 👍 
### Hint:
**In order to do this, you'd have to make your `update_comment_counter` method private**
So you'd have something like this 👇 
```
after_save :update_comment_counter

private

def update_comment_counter
  post.increment!(:commentscounter)
end
```

Reply:
Thanks a lot  for your feedback let me work on th changes